### ISSUE TITLE
Is this project still maintained?

### ISSUE THREAD

**Issue by heri16** [evanw/node-flatbuffers#2]
Is this project still maintained and working with the official binary formats?

**Comment by evanw** [comment]
No, sorry. It's not maintained as I'm no longer using flatbuffers myself.
